pkg/cacheutil/memcached_client.go

```diff
@@ -222,6 +227,7 @@ func newMemcachedClient(
 	)
 
 	c := &memcachedClient{
+		name:        name,
 		logger:      logger,
```

Comment:
Instead of adding the the name key-value pair to each log entry you can just wrap the logger with that:

```suggestion
		logger:      log.With(logger, "name", name),
```

Reply:
Yes, that's way more elegant. Thanks and updated!

Reply:
Thanks!